javaparser-core/src/main/java/com/github/javaparser/printer/ConcreteSyntaxModel.java

```diff
@@ -355,6 +355,12 @@ private static CsmElement typeArguments() {
                 child(ObservableProperty.TYPE)
         ));
 
+        concreteSyntaxModelByClass.put(PatternExpr.class, sequence(
+                child(ObservableProperty.TYPE),
```

Comment:
Here you can see one of the problems with the lexical preserving printer as it is now: shouldn't we say "comment()" in here? In what position? Dunno.

Reply:
Honestly, I don't have a clue -- at this point I'm aiming for "functional" rather than "correct" :wink: 

I suppose it might be a non-issue if any comments would be attached to the `ReferenceType` or `SimpleName` inside of the `PatternExpr` (if so, then they'll be printed when printing those?)... 

Probably needs some test cases thrown at it to figure out what the intended/actual behaviours are.